setup.cfg

```diff
@@ -61,12 +61,12 @@ testing =
 	pip>=19.1 # For proper file:// URLs support.
 	jaraco.envs>=2.2
 	pytest-xdist
-	sphinx
+	sphinx>=4.3.2
 	jaraco.path>=3.2.0
 
 docs =
 	# upstream
-	sphinx
+	sphinx >= 4.3.2
```

Comment:
This value is defined upstream, so I'll revert it. Thankfully, pull latest should be good enough.
```suggestion
	sphinx
```

Reply:
Thanks Jason!